homeassistant/components/group/__init__.py

```diff
@@ -87,35 +87,38 @@ def _conf_preprocess(value):
     extra=vol.ALLOW_EXTRA,
 )
 
-# List of ON/OFF state tuples for groupable states
-_GROUP_TYPES = [
-    (STATE_ON, STATE_OFF),
-    (STATE_HOME, STATE_NOT_HOME),
-    (STATE_OPEN, STATE_CLOSED),
-    (STATE_LOCKED, STATE_UNLOCKED),
-    (STATE_PROBLEM, STATE_OK),
```

Comment:
This line was present specifically to support the plant integration. This pull request caused regression in functionality because no group.py file was added to the plant integration. See #41492